src/portable/st/synopsys/dcd_synopsys.c

```diff
@@ -185,6 +186,40 @@ static void end_of_reset(void) {
   }
 }
 
+static void edpt_xact(uint8_t const epnum, uint8_t const dir, uint16_t const num_packets, uint16_t total_bytes) {
```

Comment:
technically this is a sub-transfer, transaction is USB terminology for **single packet data**, but we can leave it as it is now. And may do a rename later on.

Reply:
I've changed it to `edpt_schedule_packets`. This should be more clear now.